collectors/channels_collector.go

```diff
@@ -54,9 +56,12 @@ type ChannelsCollector struct {
 	// This channel should be buffered so that it does not block sends.
 	errChan chan<- error
 
+	// quit is a channel that we use to signal for graceful shutdown
+	quit chan struct{}
+
 	// cache is a map storing results from a ticker to reduce grpc server
 	// load on lnd
-	cache      map[string]interface{}
+	cache      map[string][]lndclient.ClosedChannel
```

Comment:
So this could just be `closedChannelsCache []lndclient.CllosedChannel` for simplicity since we don't store anything else in the map.